src/denoise.rs

```diff
@@ -48,13 +49,9 @@ pub struct DenoiseState {
     pitch_finder: crate::pitch::PitchFinder,
 }
 
-impl DenoiseState {
-    /// A `DenoiseState` processes this many samples at a time.
-    pub const FRAME_SIZE: usize = FRAME_SIZE;
-
-    /// Creates a new `DenoiseState`.
-    pub fn new() -> Box<DenoiseState> {
-        Box::new(DenoiseState {
+impl Default for DenoiseState {
```

Comment:
I'd intentionally made it impossible to get an on-the-stack `DenoiseState` from the public API because it's big and I wanted to avoid the potential footgun. Could you put this in a `pub(crate)` method instead of exposing it publicly?

Reply:
Sure

Reply:
Done :)